CMakeLists.txt

```diff
@@ -270,6 +270,8 @@ if(HIP_PLATFORM STREQUAL "hcc")
                 PRIVATE ${PROJECT_SOURCE_DIR}/include ${HSA_PATH}/include)
         target_link_libraries(hiprtc PUBLIC stdc++fs)
     endif()
+    target_link_libraries(hip_hcc PRIVATE /opt/rocm/lib/libamd_comgr.so)
```

Comment:
Some users, including CI machines won't install to this location.

Reply:
Better to add 
```
    if(HIP_PLATFORM STREQUAL "hcc")
        find_package(amd_comgr REQUIRED CONFIG
            PATHS
                /opt/rocm/
            PATH_SUFFIXES
                cmake/amd_comgr
                lib/cmake/amd_comgr
        )
        add_definitions(-DUSE_COMGR=1)
        MESSAGE(STATUS "Code Object Manager found at ${amd_comgr_DIR}.")
    endif()
```
Then use 
```
        target_link_libraries(hip_hcc PRIVATE amd_comgr)
        target_link_libraries(hip_hcc_static PRIVATE amd_comgr)
```

Reply:
Done